README.md

```diff
@@ -86,6 +86,16 @@ export default MyStripeCheckout;
 - `renderOnComplete` (?(props) => React$Node) - Optional rendering function returning a component to display upon checkout completion. note: You don't need this if your onSuccess and onCancel functions navigate away from the component.
 
 
+## Apple Pay and Google Pay
+- This library uses [react-native-webview](https://github.com/react-native-webview) under the hood to render the Stripe Checkout webpage. To get Apple pay and Google pay to work we need to pass the context to the browser, [here's how to get it working](https://github.com/react-native-webview/react-native-webview/issues/920#issuecomment-720305564):
+  - What causes issue - script that "injected" by default on webview start - html5HistoryAPIShimSource
+How to fix:
+    - just comment line in /node_modules/react-native-webview/apple/RNCWebView.m like shown below
+    ```
+    WKUserScript *script = [[WKUserScript alloc] initWithSource:html5HistoryAPIShimSource injectionTime:WKUserScriptInjectionTimeAtDocumentStart
+    forMainFrameOnly:YES];
+    // [wkWebViewConfig.userContentController addUserScript:script]; // this line that inject "html5HistoryAPIShimSource" on start
+    ```
```

Comment:
```suggestion
    ```

Keep in mind that this is a temporary fix that is really hard to maintain on a production application (it will break on every react-native-webview update). We need to find another solution, it's a work in progress.
```

Reply:
Please add 
in v10.9.2 line number is 1270.
This helps in finding the particular line to be commented.